### PR TITLE
fix(app): top-navigation and status bars fix

### PR DESCRIPTION
**Pull request by 32penkin**
### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/kittenTricks/blob/master/CONTRIBUTING.md) guide.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -12,6 +12,11 @@
     },
     "android": {
       "package": "com.akveo.kittentricks"
+    },
+    "androidStatusBarColor": "#3366FF",
```
Comment (artyorsh) [inline app.json:16]: what about ios?

File: src/components/messaging/chat.header.tsx [hunk 2]
```diff
@@ -60,7 +60,10 @@ class ChatHeaderComponent extends React.Component<ChatHeaderProps> {
     const { interlocutor } = this.props;
 
     return (
-      <Avatar source={interlocutor.photo.imageSource} />
+      <Avatar
+        size='medium'
```
Comment (artyorsh) [inline src/components/messaging/chat.header.tsx]: medium is default size

**Review by artyorsh — APPROVED** [review]

File: app.json (hunk 1 above)
Comment (32penkin) [reply app.json:16]: I believe we don't need such kind of adjustment for ios.

File: src/components/messaging/chat.header.tsx (hunk 2 above)
Comment (32penkin) [reply src/components/messaging/chat.header.tsx]: Refactored.

**Pull request merged**